wordpress-mobile/WordPress-FluxC-Android
# Woo Activate: Add v1.5/plans endpoint

This PR implements the REST client for the WP.com `v1.5/plans` endpoint, which provides data about all plans offered by WP.com.

**To test:**
1. Run the example app
2. Tap on the Plans button
3. Tap on the Fetch plans button
4. Verify a list of plans is listed in the output
